lstchain/tools/lstchain_fit_intensity_scan.py

```diff
@@ -362,8 +362,6 @@ def read_calibration_file(file_name, ff_data, calib_data, ped_data, tel_id=1):
     with HDF5TableReader(file_name) as h5_table:
 
         try:
-            assert h5_table._h5file.isopen == True
```

Comment:
why is this check removed?

Reply:
Because it is testing internal things of the `HDF5TableWriter`, which makes no sense in this test.

This specific test is even not possible to fail, as `HDF5TableReader` would already raise an error in `__init__` if the file could not be opened.